docs/Ways-of-Working/Onboarding/Introduction.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Introduction to Onboarding at AM
+---
+
+## Welcome to AM
+
+We're delighted to have you on board. Our structured onboarding process is designed to facilitate a smooth and enriching transition into our team.
+
+Please note, if you're in a Lead role, you'll have additional responsibilities.
```

Comment:
We should change "Lead" here (and throughout). We are referring to people joining as managers in some cases, and in others to people joining at E4+ (i.e. including Staff and Principal engs).

It may be clearer to keep this document as a summary for all engineers, and add a seperate file/s for managers and E4+. That could be framed in a way that makes it reusable for people being promoted into managerial roles. Perhaps this is a problem for later though.